test/db/SQLJSPackageDB.test.ts

```diff
@@ -0,0 +1,1003 @@
+        expect.objectContaining(valueSetFour)
+      ]);
+    });
+
+    it('should sort results using the descending order in which resources were loaded (e.g., last in first out) when sort order is ByLoadOrder descending', () => {
+      const resources = packageDb.findResourceInfos('*', { sort: [byLoadOrder(false)] });
+      expect(resources).toHaveLength(5);
+      expect(resources).toEqual([
+        expect.objectContaining(valueSetFour),
+        expect.objectContaining(valueSetThree),
+        expect.objectContaining(specialExtension),
+        expect.objectContaining(observationProfile),
+        expect.objectContaining(patientProfile)
+      ]);
+    });
+
+    it('should sort results using the ascending order of passed in types, then ascending order in which resources were loaded when sort order is ByType', () => {
+      const resources = packageDb.findResourceInfos('*', {
+        type: ['StructureDefinition', 'ValueSet'],
+        sort: [byType('StructureDefinition', 'ValueSet')]
+      });
+      expect(resources).toHaveLength(5);
+      expect(resources).toEqual([
+        expect.objectContaining(patientProfile),
+        expect.objectContaining(observationProfile),
+        expect.objectContaining(specialExtension),
+        expect.objectContaining(valueSetThree),
+        expect.objectContaining(valueSetFour)
+      ]);
+
+      // Try it both directions to ensure that options.type is not influencing order
+      const resources2 = packageDb.findResourceInfos('*', {
+        type: ['StructureDefinition', 'ValueSet'],
+        sort: [byType('ValueSet', 'StructureDefinition')]
+      });
+      expect(resources2).toHaveLength(5);
+      expect(resources2).toEqual([
+        expect.objectContaining(valueSetThree),
+        expect.objectContaining(valueSetFour),
+        expect.objectContaining(patientProfile),
+        expect.objectContaining(observationProfile),
+        expect.objectContaining(specialExtension)
+      ]);
+    });
+
+    it('should sort results using the ascending order of passed in types, then ascending order in which resources were loaded when sort order is ByType and ByLoadOrder ascending', () => {
+      const resources = packageDb.findResourceInfos('*', {
+        type: ['StructureDefinition', 'ValueSet'],
+        sort: [byType('StructureDefinition', 'ValueSet'), byLoadOrder()]
+      });
+      expect(resources).toHaveLength(5);
+      expect(resources).toEqual([
+        expect.objectContaining(patientProfile),
+        expect.objectContaining(observationProfile),
+        expect.objectContaining(specialExtension),
+        expect.objectContaining(valueSetThree),
+        expect.objectContaining(valueSetFour)
+      ]);
+
+      // Try it both directions to ensure that options.type is not influencing order
+      const resources2 = packageDb.findResourceInfos('*', {
+        type: ['StructureDefinition', 'ValueSet'],
+        sort: [byType('ValueSet', 'StructureDefinition')]
```

Comment:
It was how I manually tested FPL in early versions (before I had a bunch of unit tests and before I updated the CLI). I think it can go away now, so I'll delete it.